<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_ArchiveNavigator</class>
 <widget class="QWidget" name="w_ArchiveNavigator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <item row="1" column="1">
    <widget class="QToolButton" name="btPrev">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoRepeatInterval">
      <number>20</number>
     </property>
     <property name="arrowType">
      <enum>Qt::LeftArrow</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="btBegin">
     <property name="text">
      <string>|&lt;</string>
     </property>
     <property name="autoRepeatInterval">
      <number>94</number>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="btEnd">
     <property name="text">
      <string>&gt;|</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="3">
    <widget class="QLabel" name="labelChanelChanged">
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Прочитан:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="btMenu">
     <property name="text">
      <string>M</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QToolButton" name="btNext">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoRepeatInterval">
      <number>20</number>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="arrowType">
      <enum>Qt::RightArrow</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="labelCurrentTime">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="cbDays"/>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="btGo">
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
